.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: scala
 
+sudo: required
```

Comment:
This will cause Travis CI to use GCP instance instead of EC2. Travis CI with GCP will have larger memory. [Here](https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system) is the detailed information from Travis CI.

Reply:
Do we actually need the additional memory?

Reply:
I tried `-Xmx` with 3G, it seems without any issue. Travis CI can offer 4G when we don't use `sudo: required`. I think we don't need to use 6G. If it's fine with 3G, I will push the change later. :)